xla/service/gpu/runtime/nccl_collective_thunk.h

```diff
@@ -71,6 +71,21 @@ struct NcclCollectiveConfig {
   bool IsDegenerate(int64_t replica_count, int64_t partition_count) const;
 };
 
+template <typename T>
+absl::StatusOr<const int64_t> GetCurrentId(
```

Comment:
Rather than templating, would it make sense to impl this base on the NcclCollectiveConfig?

Reply:
after reorganizing the code turns out we don't need this util, removed.